connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorMakerConfig.java

```diff
@@ -130,15 +136,14 @@ public List<SourceAndTarget> clusterPairs() {
       */
     public MirrorClientConfig clientConfig(String cluster) {
         Map<String, String> props = new HashMap<>();
-        props.putAll(originalsStrings());
+        props.putAll(rawProperties);
```

Comment:
Why change this part? Aren't we transforming the configs later at line 141 anyways?